cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -759,8 +797,8 @@ TEST_P(ReductionParamTest, DISABLED_std_var)
 
   double var   = calc_var(int_values, int_values.size());
   double std   = std::sqrt(var);
-  auto var_agg = cudf::make_variance_aggregation(/*ddof = 1*/ ddof);
-  auto std_agg = cudf::make_std_aggregation(/*ddof = 1*/ ddof);
+  auto var_agg = cudf::make_variance_aggregation<reduce_aggregation>(/*ddof = 1*/ ddof);
+  auto std_agg = cudf::make_std_aggregation<reduce_aggregation>(/*ddof = 1*/ ddof);
```

Comment:
`ddof` is not equal to 1, it's a test parameter that varies.
```suggestion
  auto var_agg = cudf::make_variance_aggregation<reduce_aggregation>(ddof);
  auto std_agg = cudf::make_std_aggregation<reduce_aggregation>(ddof);
```